tests/suites/test_suite_psa_crypto_driver_wrappers.function

```diff
@@ -1151,6 +1151,62 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void hash_multipart_update( int alg_arg,
+                            data_t *input, data_t *hash,
+                            int forced_status_arg )
+{
+    psa_algorithm_t alg = alg_arg;
+    psa_status_t forced_status = forced_status_arg;
+    unsigned char *output = NULL;
+    psa_hash_operation_t operation = PSA_HASH_OPERATION_INIT;
+    size_t output_length;
+
+    mbedtls_test_driver_hash_hooks = mbedtls_test_driver_hash_hooks_init();
+    ASSERT_ALLOC( output, PSA_HASH_LENGTH( alg ) );
+
+    PSA_ASSERT( psa_crypto_init( ) );
+
+    /*
+     * Update none active operation, the driver shouldn't be called.
```

Comment:
Cosmetic: “none active” → “inactive” (here and in 2 other places)

Reply:
I've just done the change.